sanitizer/_synthesize_analysis.js

```diff
@@ -2,6 +2,7 @@ const _ = require('lodash');
 const text_analyzer = require('pelias-text-analyzer');
 
 const fields = {
+  'query': 'query',
```

Comment:
Can we call the param something other than query? Maybe poi_name or venue? Query is confusing I'm my opinion.